crates/typst-library/src/text/quote.rs

```diff
@@ -33,15 +33,19 @@ use crate::prelude::*;
 ///   flame of Udûn. Go back to the Shadow! You cannot pass.
 /// ]
 /// ```
-#[elem(Finalize, Show)]
+#[elem(Finalize, Show, Synthesize)]
 pub struct QuoteElem {
     /// Whether this is a block quote.
     ///
     /// ```example
-    /// #quote(attribution: [René Descartes])[cogito, ergo sum]
-    ///
-    /// #set quote(block: true)
-    /// #quote(attribution: [JFK])[Ich bin ein Berliner.]
+    /// #let descartes = quote(attribution: [René Descartes])[
+    ///   cogito, ergo sum
+    /// ]
+    /// #let jfk = quote(block: true, attribution: [JFK])[
+    ///   Ich bin ein Berliner.
+    /// ]
+    /// An inline citation would look like this: #descartes, and a block
+    /// equation like this: #jfk
```

Comment:
Why store the quotes in variables? I think this adds unnecessary complexity.

Reply:
The 80 column word wrap always made the examples hard to read, imo.

Reply:
80 columns? The examples are typically wrapped at ~37 cols to end up side-by-side.